helpers/disputable.ts

```diff
@@ -3,18 +3,20 @@ import { Agreement } from '../types/templates/DisputeManager/Agreement'
 import { Disputable, Dispute } from '../types/schema'
 import { crypto, Bytes, Address, BigInt } from '@graphprotocol/graph-ts'
 
-// appId for 'agreement.open.aragonpm.eth'
-const AGREEMENT_DISPUTE_HEADER = '34c62f3aec3073826f39c2c35e9a1297d9dbf3cc77472283106f09eee9cf47bf'
+const AGREEMENT_OPEN_APP_ID = '34c62f3aec3073826f39c2c35e9a1297d9dbf3cc77472283106f09eee9cf47bf'
+const AGREEMENT_PC_APP_ID = '15a969a0e134d745b604fb43f699bb5c146424792084c198d53050c4d08126d1'
```

Comment:
Might be worth leaving comments for the app ids :)

Reply:
added!